runtime/astar/src/lib.rs

```diff
@@ -1590,16 +1590,11 @@ pub type Executive = frame_executive::Executive<
 pub type Migrations = (Unreleased, Permanent);
 
 /// Unreleased migrations. Add new ones here:
-pub type Unreleased =
-    (pallet_dapp_staking::migration::versioned_migrations::V8ToV9<Runtime, TierSlotsArgs>,);
+pub type Unreleased = pallet_dapp_staking::migration::DappStakingCleanupMigration<Runtime>;
```

Comment:
```suggestion
pub type Unreleased = (pallet_dapp_staking::migration::DappStakingCleanupMigration<Runtime>,);
```

Reply:
make sure its consistent on all runtimes

Reply:
done - it was working on Chopsticks using the type directly